### PR TITLE
Fix osx build issues

### PR DESCRIPTION
**Pull request by tychobrailleur**
1. changes proposed in this pull request:
 
  This PR fixes a few issues preventing building the project on macOS with Gradle, and also updates the wrapper to 5.6.4.  The various commits comments explain the rationale for each change.
 
2. changelog and release_notes ...

 - [ ] have been updated
 - [x] do not require update — but happy to be convinced otherwise!

3. [Optional] suggested person to review this PR @___

@akasolace

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#Wed Apr 10 11:55:30 CEST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
```
Comment (akasolace) [inline gradle/wrapper/gradle-wrapper.properties]: I accept that commit without testing. If needed I will revert it later on.

File: build.gradle [hunk 2]
```diff
@@ -567,10 +570,10 @@ task resetDB(){
     group 'tool'
     description 'copy the database from existing install into project'
 
-    doFirst {println("Reseting DB: copying DB from `D:\\Program Files\\HattrickOrganizerDev` into  ${projectDir}")}
+    doFirst {println("Reseting DB: copying DB from `${resetDir}` into  ${projectDir}")}
 
     copy {
-        from "D:\\Program Files\\HattrickOrganizerDev\\db"
```
Comment (akasolace) [inline build.gradle]: I think no one else than me used this one before ...

File: gradlew [hunk 3]
```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```
Comment (akasolace) [inline gradlew]: I think 64bit is the default size, no ?

Comment (tychobrailleur) [reply gradlew]: I didn't change this — it's part of the new script installed by gradle wrapper.

Comment (akasolace) [reply gradlew]: ah ok
and thanks for the commits btw !

Comment (akasolace) [reply gradlew]: task fails now ...

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':HO:generatePlist'.
> No value has been specified for property 'MacAppBundlePlugin jreHome'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 1s
```

Comment (tychobrailleur) [reply gradlew]: Where do you see that failure?

Comment (akasolace) [reply gradlew]: I disapeared when I upgraded macapp graddle plugin ... seems all good now

**Pull request merged**